### ISSUE TITLE
Add optional normalization transform argument for torch hub/vision models

### ISSUE THREAD

**Issue by david-stojanovski** [pytorch/vision#8277]
### 🚀 The feature

Currently there are hard coded transforms in the torch hub models (e.g. https://github.com/pytorch/vision/blob/main/torchvision/models/inception.py) and think it would be more helpful to just have a boolean flag to select this option (maybe even user defined tuple inputs for the normalization)

### Motivation, pitch

This hopefully simple addition would allow people to much more easily use the default models with their own data. e.g. Training their own custom Inception v3 for an FID score.
